src/components/application_manager/test/app_launch/app_launch_data_json_test.cc

```diff
@@ -66,16 +67,17 @@ class AppLaunchDataJsonTest : public ::testing::Test {
  private:
   virtual void SetUp() {
     ::file_system::DeleteFile(kAppStorageFile);
-    test_last_state_ = std::unique_ptr<resumption::LastState>(
-        new resumption::LastStateImpl(kAppStorageFolder, kAppInfoStorage));
+    last_state_wrapper_ = std::make_shared<resumption::LastStateWrapperImpl>(
+        std::make_shared<resumption::LastStateImpl>("app_storage_folder",
```

Comment:
@YarikMamykin why don't you use `kAppStorageFolder` and `kAppInfoStorage`?

Reply:
@AKalinich-Luxoft fixed in https://github.com/smartdevicelink/sdl_core/pull/2626/commits/5123861995b84627338520ff72ca611a3689295a